spec/javascripts/social_networking/profile/profile-answer-controller_spec.js

```diff
@@ -24,13 +24,14 @@ describe('ProfileAnswerCtrl', function() {
       answer_text: answerText,
       profile_id: profileId,
       profile_question_id: questionId
-    };
+    }
   }));
 
   describe('#ifAnswered', function() {
     beforeEach(function() {
       controller = controller('ProfileAnswerCtrl', {
-        ProfileAnswers: function() {}
+        ProfileAnswers: function() {},
+        SN_CONSTANTS: {}
```

Comment:
If you abstract the SN_CONSTANTS out into a separate file, these empty objects wouldn't be necessary.